app/controllers/v0/benefits_claims_controller.rb

```diff
@@ -94,6 +94,33 @@ def submit5103
       render json: res
     end
 
+    def failed_upload_evidence_submissions
+      if Flipper.enabled?(:cst_show_document_upload_status, @current_user)
+        claims = service.get_claims
+        claim_ids = claims['data'].map { |claim| claim['id'].to_i }
+        # Get failed evidence submissions for all claims
+        failed_evidence_submissions = EvidenceSubmission.where(
+          claim_id: claim_ids
+        ).failed.group_by(&:claim_id)
+
+        failed_uploads = []
```

Comment:
(optional) see if there is an opportunity to abstract this failed uploads list compilation